modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientKeyValueBinaryView.java

```diff
@@ -85,6 +86,46 @@ public Tuple get(@Nullable Transaction tx, @NotNull Tuple key) {
                 Collections.emptyMap());
     }
 
+    /**
+     * Method throws UnsupportedOperationException, unconditionally.
+     *
+     * <p>Binary view doesn't support the operation because there is no ambiguity, {@code null} value means no rows found in the table for
+     * the given key.
+     */
+    @Override
+    public NullableValue<Tuple> getNullable(@Nullable Transaction tx, @NotNull Tuple key) {
+        throw new UnsupportedOperationException("Binary view doesn't allow null values. Please, use get(key) method instead.");
```

Comment:
Unmaintainable message in exception. The part about using `get(key)` could easily get not actual. I think this part should be part of JavaDoc, not the exception message.

The same is true for other methods with such a message.